classicpress-seo.php

```diff
@@ -142,14 +143,14 @@ public function __call( $name, $arguments ) {
 
 		return call_user_func_array( $name, $arguments );
 	}
-	
+
 	/**
 	 * Initialize.
 	 */
 	public function init() {
-		
+
 	}
-	
+
```

Comment:
Should we consider removing this method as we do not need it?

Reply:
Don't know why it is there, so I've left it even if does nothing.